web/frontend/src/pages/election/Show.tsx

```diff
@@ -43,11 +41,11 @@ const ElectionShow: FC = () => {
     <div className="election-details-box">
       {!loading ? (
         <div>
-          <h1>{title}</h1>
+          <h1>{electionTitle}</h1>
           <div className="election-details-wrapper">
             {isResultSet ? (
               <div className="election-wrapper-child">
-                <Result resultData={result} candidates={candidates} />
+                {/* TODO: <Result resultData={result} candidates={candidates} />*/}
```

Comment:
TODOs must be linked to Github issues so we can keep track of them and also have more context (i.e. explanation). Must then be of form `TODO: #343 (add result, blabla)`.